### ISSUE TITLE
7$: Report error on statement before first switch case

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#632]
```js
switch (true) {
  console.log("banana");  // should error, but doesn't
case true:
case false:
default:
}
```

**Comment by LeeWannacott** [comment]
![image](https://user-images.githubusercontent.com/49783296/152294522-dfe755d8-253c-4996-8ce1-8d4c488fe76a.png)
This is the error tsserver gives for this issue. Might be helpful to whom ever takes on this issue.

**Comment by singalhimanshu** [comment]
I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.

**Comment by strager** [comment]
@singalhimanshu You still have #341 claimed and incomplete.

**Comment by singalhimanshu** [comment]
> @singalhimanshu You still have #341 claimed and incomplete.

I lost track on that one, will probably redo it from scratch.
